src/utils/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+const myApi = axios.create({
+  baseURL: "https://fe-games.herokuapp.com/api",
+});
+
+export const fetchCategorySlug = (category_slug) => {
+  return myApi.get(`/reviews?category=${category_slug}`).then((res) => {
+    return res.data.reviews;
+  });
+};
+
+export const fetchReviewCard = (review_id) => {
```

Comment:
Ditto above, this fetches a singleReview, not a reviewCard, you're displaying the info on a review card